"make passwd" is broken

error message:

```
$ make passwd 
Generate Phoenix password ...
Enter a password with at least 8 characters.
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named IPython.lib
Makefile:227: recipe for target 'passwd' failed
make: *** [passwd] Error 1
```

It uses `ipython` which is not installed in the conda env.

--- a/Makefile
+++ b/Makefile
@@ -228,7 +228,7 @@ passwd: custom.cfg
 	@echo "Enter a password with at least 8 characters."
 	@bash -c "source $(ANACONDA_HOME)/bin/activate $(CONDA_ENV); python -c 'from IPython.lib import passwd; pw = passwd(algorithm=\"sha256\"); lines = [\"phoenix-password = \" + pw + \"\\n\" if line.startswith(\"phoenix-password\") else line for line in open(\"custom.cfg\", \"r\")]; file = open(\"custom.cfg\", \"w\"); file.writelines(lines); file.close()'"
 	@echo ""
-	@echo "Run \`make install restart' to activate this password."
+	@echo "Run \`make update restart' to activate this password."
 
 .PHONY: test
 test:

--- a/environment.yml
+++ b/environment.yml
@@ -15,6 +15,8 @@ dependencies:
 # pep8
   - flake8
   - pycodestyle
+# ipython used by make passwd
+  - ipython
 # phoenix
   - six
   - nginx=1.10

